From: Rahul Verma <rahul.verma@cavium.com>
Date: Wed, 26 Jul 2017 06:07:15 -0700
Subject: qed: enhanced per queue max coalesce value.
Patch-mainline: v4.14-rc1
Git-commit: 41822878b2ff40a1f57be81f1a1f0f040847b912
References: bsc#1050536 FATE#322898

Maximum coalesce per Rx/Tx queue is extended from
255 to 511.

Signed-off-by: Rahul Verma <rahul.verma@cavium.com>
Signed-off-by: Yuval Mintz <yuval.mintz@cavium.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/linux/qed/qed_if.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/qed/qed_if.h
+++ b/include/linux/qed/qed_if.h
@@ -184,7 +184,7 @@ enum qed_led_mode {
 
 #define DIRECT_REG_RD(reg_addr) readl((void __iomem *)(reg_addr))
 
-#define QED_COALESCE_MAX 0xFF
+#define QED_COALESCE_MAX 0x1FF
 #define QED_DEFAULT_RX_USECS 12
 #define QED_DEFAULT_TX_USECS 48
 
